src/docker-outside-of-docker/README.md

```diff
@@ -19,7 +19,7 @@ Re-use the host docker socket, adding the Docker CLI to a container. Feature inv
 |-----|-----|-----|-----|
```

Comment:
Can we make similar changes as per my previous comments for `docker-in-docker` Feature? Thanks!